frontend/front/src/pages/Admin/home/HomeTable.js

```diff
@@ -23,23 +22,64 @@ const getAddress = (params) => {
 const HomeTable = () => {
   const columns = [
     { field: "id", headerName: "Id", width: 50 },
+    {
+      field: "address",
+      valueGetter: getAddress,
+      headerName: "Address",
+      width: 200,
+    },
+    {
+      field: "city",
+      headerName: "City",
+      minWidth: 100,
+      maxWidth: 200,
+      flex: 1,
+    },
+    {
+      field: "zip_code",
+      headerName: "Zip Code",
+      minWidth: 100,
+      maxWidth: 150,
+      flex: 0.8,
+    },
+    {
+      field: "completed",
+      headerName: "Completed",
+      renderCell: (params) => (params.row.completed === "true" ? "Yes" : "No"),
```

Comment:
I know `completed` is a matter of debate right now but this looks good as a baseline :)